src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -1768,6 +1770,12 @@ scalar Long
 "Time using the format: `HH:MM:SS`. Example: `18:25:43`"
 scalar Time
 
+"A combination of street mode and corresponding duration"
+input DurationPerModeInput {
```

Comment:
We have a similar type, `TransportModeSlack`, so a equivalent name for the type could be `StreetModeDuration`